### PR TITLE
Census.py

### PR DESCRIPTION
**Pull request by AM123456789**

### CODE REVIEW

File: Census.py [hunk 1]
```diff
@@ -1,45 +1 @@
-import matplotlib.pyplot as plt
-import pandas as pd
-import geopandas as gpd
-from census import Census
-from us import states
-import os
 
-
-c = Census("6f46a7532e0317f2318ad51b2aa17a2643edcc72")
```
Comment (opencedar) [inline Census.py:9]: You don't want to leave your key in your code. Use environment variables. Run this in command line:
import os
key = os.getenv('MY_SECRET_KEY')

Then run:
export MY_SECRET_KEY="my-key-value"
Census.py

File: Census.py [hunk 2]
```diff
@@ -1,45 +1 @@
...
-va_census = c.acs5.state_county_tract(fields = ('NAME', 'C17002_001E', 'C17002_002E', 'C17002_003E', 'B01003_001E'),
```
Comment (opencedar) [inline Census.py:10]: What counties are these? In the future, try to make these parameters.

**Comment by opencedar** [comment]
Generally:

1. I don't know what your code does just by looking at it. Please add comments describing the various steps.
2. Please add a readme file (readme.md) describing how to use the code.

File: Census.py (hunk 1 above)
Comment (AM123456789) [reply Census.py:9]: Check now

**Comment by alamstein** [comment]
Since we are no longer using Python, I will close this PR for clarity.

**Pull request closed without merging**